v2/split_slice.go

```diff
@@ -0,0 +1,29 @@
+package pie
+
+// SplitSlice returns splited slices which length equals splitLength.
+//
+// for this [1,2,3] slice with splitLength == 2 will return [][]int{{1,2}, {3}}.
```

Comment:
nit: It's much easier to explain this with a visual table (like the test) rather than words:

```go
// Examples:
//
// 	Input       splitLength   Output
// 	[1, 2, 3]   4             [1, 2, 3]
// 	[1, 2, 3]   3             [1, 2, 3]
// 	[1, 2, 3]   2             [1, 2], [3]
// 	[1, 2, 3]   1             [1], [2], [3]
// 	[1, 2, 3]   0             panic: invalid splitLength: 0
```

Reply:
ok